backport-Guide.md

```diff
@@ -0,0 +1,84 @@
+# Kata Containers stable backport workflow
+
+* [Backport Workflow](#backport-workflow)
+
+This document provides a short guide on how to perform a stable backport of a
+PR or patchset in the Kata Containers repositories.
+
+It does not cover all eventualities that may be encountered whilst performing
+a backport, and only documents one potential git based workflow, but should
+serve as a short introduction and reminder of the basic process.
+
+## Backport Workflow
+
+The basic workflow involves creating a new local branch from the stable tree you
+are targetting, then cherry picking the commits from your master branch PR
+onto that branch. You then submit your branch to github as a PR against the
+stable branch (and not the `master` branch).
+
+The following example shows a pseudo-backport of a PR from the `master` branch
+into the `stable-1.2` branch of the `runtime` repo.
+
+- Ensure your local repo is upto date:
```

Comment:
upto => up to 
Ensure your local repo is up to date: 
Add a colon to match Line 22 on all of the following bullets, or remove the colon on Line 22 for consistency.